include/RAJA/util/Operators.hpp

```diff
@@ -425,8 +427,11 @@ struct bit_and : public detail::binary_function<Arg1, Arg2, Ret> {
   {
     return lhs & rhs;
   }
+
+RAJA_HOST_DEVICE static constexpr Ret identity() { return Ret{~0}; }
```

Comment:
I think this needs to be `~Ret{0}`. It works fine with signed type casting, but not unsigned

See this godbolt: https://godbolt.org/z/rnE9d8

Reply:
Oh, thank you, that is exactly what I needed. I'll also remember this tool.

Reply:
Implemented here: https://github.com/Parqua/RAJA/commit/acafb8aa149e8df070b9047afa71bfe2663e2328